### PR TITLE
feat(suite-native): update UI for failed txns

### LINKED ISSUE #15029: Solana - update visual of failed txn
Failed transactions are currently shown almost the same way as successful ones and it looks confusing. Update how failed txn look in:
- [x] Transaction list
- [x] Transaction detail

[Figma](https://www.figma.com/design/xSWRW8Csq7zYJ3TiqwoIxB/Bugfixes?node-id=1583-9572&node-type=section&t=wvLYlKwmzrb0rxSH-0)

Screenshot 2024-10-23 at 14 59 23
Screenshot 2024-10-23 at 15 00 17

### PR DESCRIPTION
**Pull request by vytick**
Update how failed txn look in:

- Transaction list
- Transaction detail

## Related Issue

Resolve #15029

### CODE REVIEW

File: suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx [hunk 1]
```diff
@@ -25,6 +25,15 @@ type TransactionDetailHeaderProps = {
 const ICON_SIZE = 56;
 const ICON_SPINNER_WIDTH = 3;
 
+const failedTxnTxtStyle = prepareNativeStyle<{ isFailedTx: boolean }>((_, { isFailedTx }) => ({
```
Comment (yanascz) [inline suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx]: Not sure what `TxnTxt` means, I'd go with just this:
```suggestion
const failedTxStyle = prepareNativeStyle<{ isFailedTx: boolean }>((_, { isFailedTx }) => ({
```

Same for other occurrences.

Comment (vytick) [reply suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx]: d7733272f4

**Review by yanascz — APPROVED** [review]
Seems to work fine!

**Pull request merged**